src/styleshseet/App.scss

```
@@ -84,15 +84,32 @@ select {
 }
 
 .form-container {
+	$_spacing: .7em;
 	font-family: $main-ff;
 	display: grid;
 	grid-template-columns: 3fr 5fr;
 	width: 90%;
 	gap: 1em;
 	margin: 0 auto;
```

I think something should be done about this declaration
```suggestion
	width: clamp(350px, 70vw, 800px);
```